backend/routes/users.js

```diff
@@ -4,10 +4,10 @@ const router = express.Router();
 
 /* GET users listing. */
```

Comment:
Could you add some comments here? Like an explanation for the API, you can reference my format in PR #28

Reply:
done, thanks for the pointer

Reply:
Welcome :)